_resources/port1.0/group/crossbinutils-1.0.tcl

```diff
@@ -139,6 +139,9 @@ proc crossbinutils.setup {target version} {
             ${worksrcpath}/libiberty/Makefile.in
     }
 
+    depends_build \
```

Comment:
Do we need an `-append` (to avoid removing basic stuff such as compiler on an older OS from the list)?

Reply:
(I'm asking, not yet suggesting a change. Most other portgroups are combined abd need to append, this one is a bit more special.)

Reply:
> Do we need an `-append` (to avoid removing basic stuff such as compiler on an older OS from the list)?

This won't affect compilers being added if required.

> (I'm asking, not yet suggesting a change. Most other portgroups are combined abd need to append, this one is a bit more special.)

These other Portfiles|groups have multiple subports so `-appended` is required to not overrider the common base deps.